release-notes/1.1/1.1.4.md

```diff
@@ -0,0 +1,60 @@
+# Release Notes
+
+## General Availability of Microsoft.Data.SqlClient 1.1.4 released 9 March 2021
+
+This update brings the below changes over the previous release:
+
+### Fixed
+- Fixed wrong results issues by changing the timeout timer to ensure a correct execution state [#950](https://github.com/dotnet/SqlClient/pull/950)
+- Fixed MARS header contains errors issue against .NET Framework 4.8+ [#959](https://github.com/dotnet/SqlClient/pull/959)
+
+
```

Comment:
Extra line.

Reply:
2 lines are always added before next superior heading.
(You can compare with other files)